action.yml

```
@@ -0,0 +1,36 @@
+name: "Ruff"
+description: "An extremely fast Python linter."
+author: "Ruff Contributor(s)"
+inputs:
+  options:
+    description:
+      "Options passed to Ruff. Use `ruff --help` to see available options. 
+      Default: 'check'"
+    required: false
+    default: "check"
+  src:
+    description: "Source to run ruff. Default: '.'"
+    required: false
+    default: "."
+  version:
+    description: 'Python Version specifier (PEP440) - e.g. "v0.0.259"'
```

```suggestion
    description: 'The version of ruff to use, e.g. "0.0.259"'
```

most people don't really know about python version specifier or specs behind them and i think "just copy the version you want to pin" is better advice here

---

makes sense towards usability [ unless we want to also nudge to educate ].  Accepted.

---

FYI - in my tests, "v..." also worked, it wasn't required to not have the 'v'